src/main/java/io/gravitee/repository/jdbc/management/JdbcApiKeyRepository.java

```diff
@@ -124,8 +127,8 @@ private boolean addClause(boolean first, StringBuilder query) {
     public Set<ApiKey> findBySubscription(String subscription) throws TechnicalException {
         LOGGER.debug("JdbcApiKeyRepository.findBySubscription({})", subscription);
         try {
-            List<ApiKey> apiKeys = jdbcTemplate.query("select * from " + escapeReservedWord("keys") + " where subscription = ?"
-                    , ORM.getRowMapper()
+            List<ApiKey> apiKeys = jdbcTemplate.query("select * from " + escapeReservedWord(this.tableName) + " where subscription = ?"

```

Comment:
escape the table name in the constructor
```suggestion
            List<ApiKey> apiKeys = jdbcTemplate.query(getOrm().getSelectAllSql() + " where subscription = ?"

```

Reply:
done